FRENCH/src/ch07-03-paths-for-referring-to-an-item-in-the-module-tree.md

```diff
@@ -0,0 +1,883 @@
+`eat_at_restaurant` function into a module named `customer_experience`, we’d
+need to update the absolute path to `add_to_waitlist`, but the relative path
+would still be valid. However, if we moved the `eat_at_restaurant` function
+separately into a module named `dining`, the absolute path to the
+`add_to_waitlist` call would stay the same, but the relative path would need to
+be updated. Our preference is to specify absolute paths because it’s more
+likely to move code definitions and item calls independently of each other.
+-->
+
+Choisir entre utiliser un chemin relatif ou absolu sera une décision que vous
+ferez en fonction de votre projet. Le choix se fera en fonction de si vous
+êtes susceptible de déplacer la définition de l'élément souhaité séparément ou
+en même temps que le code qui l'utilise. Par exemple, si nous déplaçons le
+module `salle_a_manger` ainsi que la fonction `manger_au_restaurant` dans un
+module qui s'appelle `experience_client`, nous aurons besoin de mettre à jour
+le chemin absolu vers `ajouter_a_la_liste_attente`, mais le chemin relatif
+restera valide. Cependant, si nous avions déplacé uniquement la fonction
+`manger_au_restaurant` dans un module `repas` séparé, le chemin absolu de
+l'appel à `ajouter_a_la_liste_attente` restera le même, mais le chemin relatif
+aura besoin d'être mis à jour. Notre préférence est d'utiliser un chemin absolu
+car il est plus facile de déplacer les codes de définitions et les appels aux
+éléments indépendamment les uns des autres.
+
+<!--
+Let’s try to compile Listing 7-3 and find out why it won’t compile yet! The
+error we get is shown in Listing 7-4.
+-->
+
+Essayons de compiler l'encart 7-3 et essayons de comprendre pourquoi il ne se
+compile pas pour le moment ! L'erreur que nous obtenons est affichée dans
+l'encart 7-4.
+
+<!--
+```text
+$ cargo build
+   Compiling restaurant v0.1.0 (file:///projects/restaurant)
+error[E0603]: module `hosting` is private
+ -- > src/lib.rs:9:28
+  |
+9 |     crate::front_of_house::hosting::add_to_waitlist();
+  |                            ^^^^^^^
+
+error[E0603]: module `hosting` is private
+  -- > src/lib.rs:12:21
+   |
+12 |     front_of_house::hosting::add_to_waitlist();
+   |                     ^^^^^^^
+```
+-->
+
+```text
+$ cargo build
+   Compiling restaurant v0.1.0 (file:///projects/restaurant)
+error[E0603]: module `accueil` is private
+ -- > src/lib.rs:9:28
+  |
+9 |     crate::salle_a_manger::accueil::ajouter_a_la_liste_attente();
+  |                            ^^^^^^^
+
+error[E0603]: module `accueil` is private
+  -- > src/lib.rs:12:21
+   |
+12 |     salle_a_manger::accueil::ajouter_a_la_liste_attente();
+   |                     ^^^^^^^
+```
+
+<!--
+<span class="caption">Listing 7-4: Compiler errors from building the code in
+Listing 7-3</span>
+-->
+
+<span class="caption">Encart 7-4 : Les erreurs de compilation à cause de la
+compilation du code de l'encart 7-3</span>
```

Comment:
Suggestion:
```markdown
Les erreurs de compilation du code de l'encart 7-3
```

Reply:
Exactly, it's more concise and more natural ! :+1: